src/main/java/org/prebid/server/bidder/yahoossp/YahooSSPBidder.java

```diff
@@ -144,6 +166,39 @@ private static Banner modifyBanner(Banner banner) {
                 .build();
     }
 
+    private Regs modifyRegs(Regs regs) {
+        Integer gdpr = regs.getGdpr();
+        String usPrivacy = regs.getUsPrivacy();
```

Comment:
Pls use final keywords for gdpr usPrivacy variables
And make methods like `resolveGdpr(regsGdpr, extRegsGdpr)` 
and `resolveUsPrivacy(regsUsPrivacy, extUsPrivacy)`

Reply:
I moved the extraction logic in separate methods.